content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+Continuous integration automatically rebuilds and deploys the manuscript when it changes [@doi:10.1038/nbt.3780; @doi:10.1038/550143a].
+Manubot provides many state-of-the-art features.
+For example, users cite standard identifiers, while bibliographic details are automatically retrieved.
+
+Currently, the Manubot consists of two repositories: [`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock) (a blank manuscript with the required files) and [`greenelab/manubot`](https://github.com/greenelab/manubot) (a backend Python package).
+As of December 2017, [10](https://github.com/greenelab/manubot-rootstock/graphs/contributors) individuals have contributed to the **Manubot codebase**, which has [33](https://github.com/greenelab/manubot-rootstock/stargazers) ⭐s on GitHub.
+For more information on the Manubot, see [these slides](http://slides.com/dhimmel/manubot#/) as well as the in-progress [Meta Review](https://greenelab.github.io/meta-review/).
+
+We initially created the Manubot to enable the **Deep Review**, a review paper on deep learning that we collaboratively drafted [on GitHub](https://github.com/greenelab/deep-review).
+Over 30 authors contributed to the Deep Review [@doi:10.1101/142760], using [issues](https://github.com/greenelab/deep-review/issues?utf8=%E2%9C%93&q=is%3Aissue) to discuss source material and [pull requests](https://github.com/greenelab/deep-review/pulls?utf8=%E2%9C%93&q=is%3Apr) to propose manuscript changes.
+The community appreciated the breadth of perspectives on a quickly-developing topic and the Deep Review was the most-viewed _bioRxiv_ preprint from April–June 2017 [@doi:10.1038/d41586-017-08493-x].
+We've also used the Manubot to write our [Sci-Hub Coverage Study](https://greenelab.github.io/scihub-manuscript/) and to reproduce the [Bitcoin Whitepaper](https://dhimmel.github.io/bitcoin-whitepaper/) [@tag:steemit-post].
+
+We are seeking support to fund Manubot development in 2018.
+Some **specific aims** that we're excited about are:
+
+1. JATS is the predominant archival format for scholarly manuscripts that is optimized for machine-readability and interoperability.
+We want the Manubot to **export to JATS** and to incorporate an existing **JATS viewer**, such as [Lens](https://lens.elifesciences.org/about/), into Manubot webpages.
+
+2. Currently all changes to a manuscript's source are tracked using git.
+However, Manubot does not automatically generate **diffs for rendered outputs**.
+By creating rendered diffs we can show how a manuscript changed from the reader's perspective and make creating "tracked changes" documents for journal resubmission easy.
+
+3. **End-to-end reproducibility** is when the provenance of every manuscript result can be traced to its source.
+Manubot allows users to pass template variables, so results can be embedded directly from the analysis that created them.
```

Comment:
> We will continue to evaluate and improve the Manubot template variables functionality, which allows users to embed results and method parameters directly from the analysis that created them.

Reply:
Done in https://github.com/greenelab/manufund-2018/pull/2/commits/ffd78fb31da0c87379651eb054df82425a3026ec